.github/workflows/comments.yml

```diff
@@ -119,6 +121,39 @@ jobs:
               if (!$commentToPost -and !$labelToAdd) {
                 Write-Host "Neither 'commentToPost' nor 'labelToAdd' was set, so not notifying anybody"
               }
+
+              # this is where check if a comment id is set, if so call minimize comment here
+              if ($commentId) {
```

Comment:
We probably also want some setting on the yaml template like `autoMarkAbuse` (or some better name 😄), so it would only run the code below if you opt into it.